static/src/javascripts/projects/common/views/ui/notifications-subscribe-link.html

```diff
@@ -0,0 +1,9 @@
+<div class="js-notification-link notifications-link hidden-notifications-link">
+    <a class="js-notifications-subscribe-link button button--follow"><%= text %></a>
```

Comment:
This should be a `button` instead of an `a` because it doesn't link to another page. Making it a `button` will make it  accessible to screen readers and keyboard users :-)